rust/lance-file/src/v2/reader.rs

```diff
@@ -57,6 +57,18 @@ pub struct BufferDescriptor {
     pub size: u64,
 }
 
+#[derive(Debug)]
+pub struct FileStatistics {
+    pub columns: Vec<ColumnStatistics>,
+}
+
+// Currently we report the `num_pages` and `size_bytes` of each column in Lance file.
+#[derive(Debug)]
+pub struct ColumnStatistics {
+    pub num_pages: usize,
+    pub size_bytes: u64,
+}
```

Comment:
```suggestion
/// Statistics summarize some of the file metadata for quick summary info
#[derive(Debug)]
pub struct FileStatistics {
    /// Statistics about each of the columns in the file
    pub columns: Vec<ColumnStatistics>,
}

/// Summary information describing a column
#[derive(Debug)]
pub struct ColumnStatistics {
    /// The number of pages in the column
    pub num_pages: usize,
    /// The total number of data & metadata bytes in the column
    ///
    /// This is the compressed on-disk size
    pub size_bytes: u64,
}
```

Let's document these since they are public members.

Reply:
thanks, suggestion applied.